css/css-will-change/will-change-stacking-context-001.html

```diff
@@ -1,36 +0,0 @@
-<!doctype html>
-<meta charset="utf-8">
-<title>CSS Test: will-change: filter should create a stacking context.</title>
-<link rel="author" title="Philip Rogers" href="mailto:pdr@chromium.org">
-<link rel="help" href="https://bugs.chromium.org/p/chromium/issues/detail?id=960953">
-<link rel="help" href="https://drafts.csswg.org/css-will-change/#will-change">
-<link rel="help" href="https://drafts.fxtf.org/filter-effects/#FilterProperty">
-<link rel="match" href="will-change-stacking-context-001-ref.html">
-<style>
-  .indicator {
-    position: absolute;
-    background-color: green;
-    z-index: 1;
-  }
-  .willchange {
-    will-change: filter;
-    z-index: 0;
-  }
-  .child {
-    position: relative;
-    background-color: red;
-    z-index: 2;
-  }
-  .box {
-    width: 100px;
-    height: 100px;
-  }
-</style>
-
-<div class="indicator box"></div>
-<div class="willchange box">
-  <!-- Because will-change: filter creates a stacking context, this child
-       remains on bottom even though it has a higher z-index than the indicator
-       box. -->
-  <div class="child box"></div>
```

Comment:
Why are these removed? Are they already in the css-will-change subdir?

Reply:
It's testing the same thing as will-change-stacking-context-filter-1.html